File: Python PowerPoint Examples/ShapesApiDemo/setParagraphTextRangeProperties.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.shapes_api.ShapesApi(configuration)

fileName = "SetParagraphTextRangeProperties.pptx"
password = ""
storage = ""
inputFolder = "input"
slide_index = 0
shape_index = "0"
paragraph_index = 0
portion_index = 0
source = spirecloudpowerpoint.models.Portion()
source.text = "SetParagraphTextRangeProperties"
source.font_height = 20
source.font_color = "green"
source.spacing = 2
source.font_bold = True
result =api.set_paragraph_text_range_properties(fileName, slide_index, shape_index, paragraph_index, portion_index,source=source, password=password, storage=storage,folder=inputFolder)

File: JavaScript PowerPoint Examples/ShapesApiDemo/setParagraphTextRangeProperties.js
(function (){
    var appId = "your ID";
    var appKey = "your Key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudPowerpoint = require('../../src/index');
    var apiClient = new SpirecloudPowerpoint.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudPowerpoint.ShapesApi(apiClient);
    
    var name = 'SetParagraphTextRangeProperties.pptx';
    var folder = 'input';
    var slideIndex = 0;
    var shapeIndex = 0;
    var paragraphIndex = 0;
    var portionIndex = 0;
    var opts = {};
    opts.source = new SpirecloudPowerpoint.Portion();
    opts.source.Text = "SetParagraphTextRangeProperties";
    opts.source.FontBold = true;
    opts.source.Spacing = 2.0;
    opts.source.FontHeight = 20.0;
    opts.source.FontColor = "green";
    opts.folder = folder;
    var response = instance.setParagraphTextRangeProperties(name, slideIndex, shapeIndex, paragraphIndex, portionIndex, opts,
     function (error, data, response) {
        if (error) throw error;
     });
})();